makefile

```diff
@@ -77,12 +77,28 @@ codeall: code codetest
 doc: docstyle
 
 # Testing
-test:
+test: test-backend test-frontend
+test-backend:
 	python -m unittest discover -v
 testdoc:
 	python -m test.test --doctests-only
 testall: test testdoc
 
+## Testing - Frontend
+## - ENVIRONMENTS: To run multiple, hyphen-delimit, e.g. ENVIRONMENTS=local-dev-prod
+ENVIRONMENTS=ENVIRONMENTS=local
```

Comment:
@Sigfried I parameterized the test script, so that it's set up to run just `local` by default locally, and `dev` and/or prod for now in the GitHub action.

You had [some ideas](https://github.com/jhu-bids/TermHub/issues/523#issuecomment-1690265080) on implementation, but I looked at the [playwright docs](https://playwright.dev/docs/test-parameterize) and they had a standard way of doing it with environmental variables so I did it that way.